### ISSUE TITLE
Change package declaration QF breaks calls to URI extension method

### ISSUE THREAD

**Issue by tivervac** [eclipse/xtext-xtend#514]
If you apply the "Change package declaration" QF on the first line `e.URI` becomes `e.uRI`
```
package obviously.not.the.correct.pkg.name

import org.eclipse.emf.ecore.EObject

class Test {
	
	def void doThing() {
		val EObject e = null
		e.URI
	}
	
	def void getURI(EObject e){}
}
```

**Comment by cdietrich** [comment]
Fixed in 2.15

**Comment by cdietrich** [comment]
thx @tivervac
